collection-scripts/gather_serving

```diff
@@ -2,6 +2,12 @@
 # shellcheck disable=SC1091
 source common.sh
 resources=("inferenceservices" "inferencegraphs" "trainedmodels" "servingruntimes" "clusterstoragecontainers" "predictors")
+# Dependent resources ossm
+resources+=("smcp" "smm" "smmr" "gateways" "authorizationpolicies" "envoyfilters")
```

Comment:
The `virtualservice` is missing. Additionally, it is not possible to gather the secret, correct? This is because we have TLS secrets in place.

Reply:
we do not collect secret by default, i wil add virtualservices in the list